src/ecrc-frontend/src/components/base/declaration/Declaration.css

```diff
@@ -0,0 +1,33 @@
+.declareTitle {
+  width: 100%;
+  height: 36px;
+  background: inherit;
+  background-color: rgba(56, 89, 138, 1);
+  box-sizing: border-box;
+  border-width: 3px;
+  border-style: solid;
+  -moz-box-shadow: none;
```

Comment:
do we need to provide these properties if they're just being set to `none`?

Reply:
ok. will remove it.